python/pyarrow/parquet/core.py

```diff
@@ -191,6 +190,11 @@ def convert_single_predicate(col, op, val):
     return reduce(operator.or_, disjunction_members)
 
 
+_filters_to_expression = _deprecate_api(
```

Comment:
The _deprecate_api is not imported

Reply:
Ah, and I thought I wouldn't need to run the tests locally after dropping that commit, "BeCaUsE tHiS wAs EaSy" :stuck_out_tongue_winking_eye: